### DISCUSSION TITLE
blog/v2-stories

### DISCUSSION THREAD

**Discussion by giscus** [zenstackhq/zenstack-docs#300] [Comments]
# blog/v2-stories

the main features of ZenStack V2 and the stories behind it.

https://zenstack.dev/blog/v2-stories

**Comment by octadecimal** [comment]
This is an amazing post; it's clear just how much passion has gone into this project. I loved reading every word of it.

Congratulations on the release, and THANK YOU for the wonderful tool!

**Comment by Eliav2** [comment]
I genuinely think zenstack is one the best fullstack libs out there. It's battery included with type safe code generation make it's easy to iterate development cycles. And it's even have well documented plugins system. Your team are highly skilled and devoted and each issue gets special attention.

Just amazing guys. Thank you for the awesome work and keep up!
